mcs/tools/gacutil/gacutil.exe.sources

```diff
@@ -1,2 +1,9 @@
 driver.cs
 ../security/StrongNameManager.cs
+../../../external/ikvm/reflect/*.cs
+../../../external/ikvm/reflect/Emit/*.cs
+../../../external/ikvm/reflect/Metadata/*.cs
```

Comment:
Some of these might not be needed, which could be used to remove the System/Mono.CompilerServices.SymbolWriter deps as well.